pkg/controller/quay_io_ci_images_distributor/mirror.go

```diff
@@ -20,6 +20,7 @@ type MirrorTask struct {
 	CurrentQuayDigest string                                `json:"current_quay_digest"`
 	CreatedAt         time.Time                             `json:"created_at"`
 	Stale             bool                                  `json:"stale"`
+	Description       string                                `json:"description"`
```

Comment:
What type of description will this be?

Reply:
Currently it is used to store information for debugging.
There is an endpoint from the server to show the current content of the queue.
This value is only used there.

Reply:
Changed it to `owner` for better readability.